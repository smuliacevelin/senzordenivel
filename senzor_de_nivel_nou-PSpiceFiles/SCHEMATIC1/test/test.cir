** Profile: "SCHEMATIC1-test"  [ c:\users\evelin\documents\facultate\an 2 sem 2\cad\senzor_de_nivel_nou-pspicefiles\senzor_de_nivel_nou-PSpiceFiles\SCHEMATIC1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\SPB_DATA\cdssetup\OrCAD_PSpice\17.4.0\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.DC LIN PARAM Rx 20k 30k 100 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
