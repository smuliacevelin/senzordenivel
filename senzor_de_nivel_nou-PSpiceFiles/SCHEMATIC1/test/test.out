
**** 05/13/22 18:59:54 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ c:\users\evelin\documents\facultate\an 2 sem 2\cad\senzor_de_nivel_nou-pspicefiles\senzor_de_nivel_


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\SPB_DATA\cdssetup\OrCAD_PSpice\17.4.0\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.DC LIN PARAM Rx 20k 30k 100 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source SENZOR_DE_NIVEL_NOU
R_R2         0 N30894  {Rx} TC=0,0 
X_U3A         N30628 N30930 VCC -VCC N30930 LM358
R_R21         N31366 N31354  15k TC=0,0 
X_U6B         N31856 N31767 VCC -VCC N31954 LM358
X_U6A         N30894 N30898 VCC -VCC N30906 LM358
R_R3         0 N30894  150k TC=0,0 
R_R14         N30620 N30628  1k TC=0,0 
V_V6         N31366 0 16V
R_R8         0 N30522  1k TC=0,0 
V_V2         N30418 0 16V
R_R23         N31346 N31378  1k TC=0,0 
R_R24         0 N31378  1k TC=0,0 
R_R10         N30636 N30580  1k TC=0,0 
X_U5B         N30918 N30620 VCC -VCC N30628 LM358
R_R1         N30894 N30672  100k TC=0,0 
R_R17         N30930 N31172  1k TC=0,0 
R_R25         N31767 VCC  15k TC=0,0 
R_R7         N30522 N30418  5.6k TC=0,0 
X_U4B         N30522 N30636 VCC -VCC N30580 LM358
R_R22         0 N31354  1k TC=0,0 
X_U8A         N31172 N31378 VCC -VCC N31346 LM358
V_V1         N30672 0 16V
R_R15         0 N30918  1k TC=0,0 
V_V4         -VCC 0 -16V
V_V3         VCC 0 16V
R_R13         N30906 N30918  180 TC=0,0 
R_R11         N30898 N30906  1k TC=0,0 
R_R9         N30898 N30636  1k TC=0,0 
R_R16         N31184 N31172  1k TC=0,0 
R_R26         0 N31767  1k TC=0,0 
R_R12         N30580 N30620  180 TC=0,0 
X_U7B         N31354 N31184 VCC -VCC N31184 LM358
R_R27         N319821 N31954  1k TC=0,0 
X_U8B         N31856 N32924 VCC -VCC N32145 LM358
R_R28         N34393 N32145  1k TC=0,0 
X_U9B         N31856 N32449 VCC -VCC N32433 LM358
R_R31         N34436 N32433  1k TC=0,0 
R_R34         N32924 VCC  2.4k TC=0,0 
R_R35         0 N32924  1k TC=0,0 
R_R37         0 N32449  1.2k TC=0,0 
R_R36         N32449 VCC  1k TC=0,0 
R_R38         N33354 VCC  1k TC=0,0 
R_R39         0 N33354  2k TC=0,0 
X_U10B         N31856 N33354 VCC -VCC N33560 LM358
R_R40         N34630 N33560  1k TC=0,0 
D_D1         N319821 N32145 Dbreak 
D_D2         N34393 N32433 Dbreak 
D_D3         N34436 N33560 Dbreak 
D_D4         N34630 0 Dbreak 
.PARAM  rx=20k

**** RESUMING test.cir ****
.END

**** 05/13/22 18:59:54 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ c:\users\evelin\documents\facultate\an 2 sem 2\cad\senzor_de_nivel_nou-pspicefiles\senzor_de_nivel_


 ****     Diode MODEL PARAMETERS


******************************************************************************




               Dbreak          X_U3A.x_lm358.dx 
          IS   10.000000E-15  800.000000E-18 
          RS     .1             1            
         CJO  100.000000E-15                 


               X_U3A.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


               X_U6B.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U6B.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


               X_U6A.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U6A.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


               X_U5B.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U5B.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


               X_U4B.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U4B.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


               X_U8A.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U8A.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


               X_U7B.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U7B.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


               X_U8B.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U8B.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


               X_U9B.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U9B.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


               X_U10B.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U10B.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


**** 05/13/22 18:59:54 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ c:\users\evelin\documents\facultate\an 2 sem 2\cad\senzor_de_nivel_nou-pspicefiles\senzor_de_nivel_


 ****     BJT MODEL PARAMETERS


******************************************************************************




               X_U3A.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


               X_U6B.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


               X_U6A.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


               X_U5B.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


               X_U4B.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


               X_U8A.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


               X_U7B.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


               X_U8B.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


               X_U9B.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


               X_U10B.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


**** 05/13/22 18:59:54 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ c:\users\evelin\documents\facultate\an 2 sem 2\cad\senzor_de_nivel_nou-pspicefiles\senzor_de_nivel_


 ****     Current Controlled Switch MODEL PARAMETERS


******************************************************************************




               X_U3A.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U3A.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U3A.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U3A.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U6B.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U6B.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U6B.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U6B.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U6A.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U6A.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U6A.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U6A.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U5B.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U5B.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U5B.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U5B.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U4B.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U4B.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U4B.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U4B.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U8A.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U8A.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U8A.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U8A.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U7B.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U7B.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U7B.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U7B.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U8B.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U8B.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U8B.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U8B.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U9B.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U9B.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U9B.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U9B.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U10B.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U10B.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U10B.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U10B.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 



          JOB CONCLUDED

**** 05/13/22 18:59:54 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ c:\users\evelin\documents\facultate\an 2 sem 2\cad\senzor_de_nivel_nou-pspicefiles\senzor_de_nivel_


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .25
